app/lib/court_reminders_importer.rb

```diff
@@ -0,0 +1,52 @@
+module CourtRemindersImporter
+  def self.generate_reminders(court_dates, court_locations)
+    time_zone_offset = '-0600' # Make sure this is by instance eventually
+
+    Message.transaction do
+      Message.scheduled.auto_court_reminders.destroy_all
+      court_dates.each do |court_date|
+        matching_rrs = ReportingRelationship.where(notes: court_date['ofndr_num'], active: true)
+        Rails.logger.info { "[importing court reminders] Found #{matching_rrs.count} RRs with IDs #{matching_rrs.pluck(:id)} with the same ctrack" }
+
+        rr = matching_rrs.left_joins(:messages).order('messages.send_at DESC NULLS LAST').first
+        next if rr.nil?
+
+        court_date_at = Time.strptime("#{court_date['crt_dt']} #{court_date['crt_tm']} #{time_zone_offset}", '%m/%d/%Y %H:%M %z')
+        next if court_date_at < Time.zone.now
```

Comment:
Should this maybe be `< Time.zone.now + 1.day`? Not sure. Is the `send_at = ...` on line 25 designed to fail validation if `court_date_at` is less than `Time.zone.now + 1.day` and take care of edge cases?